polaris-react/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -70,6 +73,7 @@ export function MenuGroup({
       accessibilityLabel={accessibilityLabel}
       onClick={handleClick}
       getOffsetWidth={handleOffsetWidth}
+      activatorRef={groupActivatorRef}
```

Comment:
Allows to return activator to the group of actions 

https://user-images.githubusercontent.com/104942025/226448012-05c7a937-c64c-43e9-8dff-ba21e55ad83a.mov